indico/web/client/js/custom_elements/ind_with_tooltip.js

```diff
@@ -24,7 +24,6 @@ customElements.define(
     hide() {
       clearTimeout(this.timer);
       this.shown = false;
-      window.removeEventListener('resize', this.updatePosition);
```

Comment:
Do you remember by any chance what the reason for this seemingly unrelated change was?

Reply:
Honestly, no. 😂 From the looks of it, it was probably not needed to begin with (I don't see that the listener is added somewhere), so I might have gone "Hmm, what's this doing here??" but then forgot to split the change into a separate branch.